README.md

```diff
@@ -18,6 +18,8 @@
 
 [![MaxText Package Tests](https://github.com/AI-Hypercomputer/maxtext/actions/workflows/RunTests.yml/badge.svg)](https://github.com/AI-Hypercomputer/maxtext/actions/workflows/build_and_test_maxtext.yml)
 
+> **_NOTE:_** We recommend running MaxText with Python 3.12, as it is our primary supported version. Other Python versions may encounter compatibility issues.
+
```

Comment:
Can you also remove the equivalent note added in https://github.com/AI-Hypercomputer/maxtext/pull/2663? https://github.com/AI-Hypercomputer/maxtext/pull/2517 is going to directly pull this main README contents into ReadTheDocs